install/preview/prettylog/main.go

```diff
@@ -38,6 +38,20 @@ var (
 )
 
 func main() {
+	// Warn and wait for user approval
+	pterm.FgBlue.Println(`
+Welcome to the local preview of Gitpod. Please note the following limitations:
```

Comment:
I'd suggest we change this to: 
```
Welcome to the local preview of Gitpod. Please note the following limitations:
  - Performance is limited by the capabilities of your machine - a minimum of 4 cores and 6GB of RAM are required
  - ARM CPUs including Macs with Apple Silicon (e.g. M1) are currently not supported
  For more information about these limitation, please visit the [local preview documentation](https://www.gitpod.io/docs/self-hosted/latest/local-preview).
```
--> This adds a link & removes the periods at the end of the bullet points.
--> I am assuming we can do links in the CLI output here, but not sure how / if this works.

Reply:
<img width="1450" alt="Screenshot 2022-09-21 at 11 31 13 AM" src="https://user-images.githubusercontent.com/7892868/191426111-2cdb7239-bc33-454b-b1da-761e57ed3ae0.png">


It is rendered like this, with the http link being clickable (depends on the shell too)

Reply:
In that case we should likely not try to make the http link a clickable link (with the [...](...) ) like on the web but instead have the link as full text: 
```
Welcome to the local preview of Gitpod. Please note the following limitations:
  - Performance is limited by the capabilities of your machine - a minimum of 4 cores and 6GB of RAM are required
  - ARM CPUs including Macs with Apple Silicon (e.g. M1) are currently not supported
  For more information about these limitation, please visit the local preview documentation: https://www.gitpod.io/docs/self-hosted/latest/local-preview

Reply:
Updated.